index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rock Scissors Paper</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Audiowide&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css" type="text/css" />\
+</head>
+<body>
+    <div class="main-wrapper">
+        <div class="content-wrapper">
+            <header>
+                <h1>ROCK SCISSORS PAPER</h1>
+            </header>
+
+            <div class="round">
+                <p id="round-number">Round: 1</p>
+            </div>
+
+            <div class="game">
+                <div class="player">
+                    <p id="user-name">User's name</p>
+                    <p class="points">Points: <b id="user-points">0</b></p>
+
+                    <div class="images">
+                        <img src="img/paper.png" alt="paper" height="170px" width="170px">
```

Comment:
jpg has alwyas background - we should find `svg` or `png` - eventually - you are happy with current solution